src/components/Filters.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+const Filters = ({ products }) => {
+
+  const createFilters = () => {
+    let filters = {};
+
+    products.forEach((product) => {
+      Object.keys(product.attributes).forEach((attr) => {
+
+        // Add keys
+        if (!filters[attr]) {
+          filters[attr] = [];
```

Comment:
I don't love the pattern of defining a variable, and then modifying it somewhere else. I can't exactly explain why, but it feels scary. Anyway, I like to use `Array.prototype.reduce` for this kind of thing: you're taking an array and reducing it to a single value (an object). Try to rewrite it that way.